Dockerfile

```diff
@@ -1,33 +1,58 @@
 FROM python:3.6.5-jessie
 
 env DEBIAN_FRONTEND noninteractive
-RUN apt-get update && \
-    apt-get -y install \
+RUN echo 'deb http://deb.debian.org/debian jessie-backports contrib non-free main' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y install --no-install-recommends \
+            build-essential \
+            chromedriver \
+            chrpath \
             dbus-x11 \
+            ffmpeg \
             libdbus-1-dev \
+            libdbus-glib-1-dev \
```

Comment:
You start from python image with version 3.6.5 installed but this line install python3.5 as dependency. You could start from debian instead of python if you do not want to have several python installed (increase size for nothing). You can check that with 
`apt install --dry-run  libdbus-glib-1-dev`